en/development/testing.rst

```diff
@@ -1195,13 +1195,18 @@ and make sure our web service is returning the proper response::
                 ['id' => 1, 'lng' => 66, 'lat' => 45],
             ];
             $expected = json_encode($expected, JSON_PRETTY_PRINT);
-            $this->assertEquals($expected, (string)$this->_response->getBody());
+            $this->assertEquals($expected, $this->_response->getStringBody());
```

Comment:
Should we be telling folks to use `assertResponseEquals()` instead?

Reply:
We could add that on top maybe? But some need the string itself for partial comparison or processing.